.github/labels.js

```diff
@@ -94,7 +94,7 @@ module.exports = [
     {
         name: "waiting-for-feedback",
         color: "ffffff",
-        description: "Waiting for 4 weeks until issues or pull request will be closed"
+        description: "Waiting for 90 days until issues or pull request will be closed"
```

Comment:
4 weeks are not 90 days and below you write "- will be marked as stale after 60 days - will be closed after next 30 days"

Typo?

Reply:
finally will be closed after 90 days, intermediate step as marked as stale is not important here